### DISCUSSION TITLE
API in Container App behind Application gateway

### DISCUSSION THREAD

**Discussion by mnoconexus** [Azure/data-api-builder#2229] [Q&A]
Hi!

I have an Container app with Vnet integration and sitting behind an Application Gateway.  The Application URL is like this xxxx.azurecontainerapps.io , but from the outside i have a completly different url. How do I get the nextLink on the REST result to have the correct URL in it and not the  xxxx.azurecontainerapps.io  URL?  Is there an Enviroment setting that can be used?

**Comment by sijpesteijn** [comment]
We are facing the same problem. A suggestion would be to create a relative nextLink, so without the hostname.

**Comment by mirokauppinen** [comment]
I have the same problem. I'm new to Container apps and doing a PoC with DAB. I have tried to mess with the CA setups but I think this cannot be changed from the container app.  
I have 2 suggestions
Either the hostname could be part of the config file or remove the host from the nextLink.

**Comment by michsob** [comment]
Is there any solution for this already?
